docs/ACCESSIBILITY.md

```diff
@@ -127,6 +129,61 @@ When the "New messages" separator is activated, it logically moves the separator
 
 Lastly, we style the "New messages" separator like a normal button, styled it to float on top of the transcript, and added `click` and `keypress` event listener for <kbd>ENTER</kbd> and <kbd>SPACEBAR</kbd> key for [activation](https://www.w3.org/TR/wai-aria-practices-1.1/#button).
 
+## UX: Message ordering
+
+### User story
+
+Azure Bot Services is a distributed system and message order is not guaranteed. Web Chat use insertion sort based on the timestamp to order messages.
+
+Messages with a latter timestamp may arrive sooner than messages with a former timestamp. Thus, messages with latter timestamp could appear on the screen first. Then, messages with a former timestamp will get inserted before it.
```

Comment:
```suggestion
Messages with a newer timestamp may arrive before messages with an older timestamp. Thus, messages with a newer timestamp could appear on the screen first. Then, messages with an older timestamp will get inserted before it.
```

Reply:
Thank you. This is always something I struggle when reading English: why sometimes people use former/latter? Is it only good for referencing things in the current article? E.g. "the former section told you ABC, the latter section told you XYZ."

Reply:
Yea, typically 'former' and 'latter' refer to something previously mentioned.